src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -141,4 +141,25 @@ public String getDisplayNameGiver() {
     public String getDisplayNameRecipient() {
         return displayNameRecipient;
     }
+    
+    /**
+     * @return A user-friendly {@FeedbackParticipantType} representing participant list.
+     */
+    public static List<FeedbackParticipantType> getParticipantListFromParams(
+            String getParticipantListFromCommaSeparatedValues) {
```

Comment:
It looks like I mixed up the param name and method name in my review. My bad. What I meant was `getParticipantListFromCommaSeparatedValues` as the method name (but the name `get*` should suggest that, right?). The parameter name can correspondingly be `commaSeparatedValues`.